cmd/sops/subcommand/updatekeys/updatekeys.go

```diff
@@ -66,11 +66,22 @@ func updateFile(opts Opts) error {
 			keysWillChange = true
 		}
 	}
-	if !keysWillChange {
+
+	// TODO: use conf.ShamirThreshold instead of tree.Metadata.ShamirThreshold in the next line?
+	//       Or make this configurable?
+	var shamirThreshold = tree.Metadata.ShamirThreshold
+	if opts.GroupQuorum != 0 {
```

Comment:
A problem in [#1509](https://github.com/getsops/sops/pull/1509), was that the argument `shamir-secret-sharing-quorum` was not allowed for the command, and therefore the `Opts.GroupQuorum`was always 0 (zero). In this branch, the issue does not seem to have been resolved (and if fixed, it should probably be renamed, e.g to `shamir-secret-sharing-threshold`).

Reply:
That has already been fixed in #1608. This branch is only for showing the changes, not for anything else.

Reply:
Ah, beautiful! 😃 Did not spot that one. I'll 🤐